python/examples/tornado_handler_server.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import absolute_import
+
+import argparse
+import collections
+import contextlib
+import cProfile
+import itertools
+import logging
+import pstats
+import sys
+
+import tornado.ioloop
+import tornado.web
+from tchannel.tornado import TChannel
+from tchannel.tornado.tornado_req_handler import TornadoRequestHandler
+
+
+log = logging.getLogger('tchannel')
+
+
+def parse_args(args=None):
+    args = args or sys.argv[1:]
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "-v", "--verbose",
+        dest="verbose",
+        action="store_true"
+    )
+
+    parser.add_argument(
+        "--listen",
+        dest="in_port",
+        default=None,
+        type=int,
+        help="Port for inbound connections"
+    )
+
+    args = parser.parse_args(args)
+    return args
+
+
+class MainHandler(tornado.web.RequestHandler):
+    def get(self):
```

Comment:
Are you sure you want to do this? `tornado.web` seems like it's coupled with HTTP.